### ISSUE TITLE
Comments are incorrectly reported by PSR2.ControlStructures.SwitchDeclaration sniff

### ISSUE THREAD

**Issue by OndraM** [squizlabs/PHP_CodeSniffer#683]
Hi,
since 2.3.3 (and caused probably by #621) comments are incorrectly reported as an error in following structure:

```php
switch ($foo) {
    case 'bar': // some comment
        ...
        break;
    case 'baz': // other comment
        ...
        break;
    default: // other comment
        ...
        break;
}
```

On the lines with the comment following error occurs:
`ERROR | [x] The CASE body must start on the line following the statement`

... though the case body actually starts on a proper line.

**Comment by gsherwood** [comment]
Thanks for reporting.

**Comment by rmbl** [comment]
There still seems to be a problem with comments in the next line after the case statement, which triggers errors for our codebase.

```php
switch ($foo) {
    case 'bar': 
        // some comment
        ...
        break;
}
```

**Comment by gsherwood** [comment]
Thanks again. I think I've got the problem this time.
